daemons/controld/controld_callbacks.c

```diff
@@ -196,6 +196,7 @@ peer_update_callback(enum crm_status_type type, crm_node_t * node, const void *d
             }
 
             if (!appeared) {
+                node->peer_lost = time(NULL);
```

Comment:
Does this need to get cleared somewhere, perhaps when the peer re-appears?

Reply:
No, we want to know the last time it was lost, regardless of whether it came back or not. The peer might go away and come back quickly, we only care about it going away